models/tracker/src/migration.ts

```diff
@@ -386,6 +398,10 @@ async function migrateIssueStatuses (client: MigrationClient): Promise<void> {
 export const trackerOperation: MigrateOperation = {
   async preMigrate (client: MigrationClient, logger: ModelLogger): Promise<void> {
     await tryMigrate(client, trackerId, [
+      {
```

Comment:
Minor: since you are now updating both `Status` and `Model` domains the order should not matter that much. Maybe just to be consistent with other workspaces where it was physically applied earlier. (So nothing to do here just an observation).